lang/fr/gallery.php

```diff
@@ -12,231 +12,231 @@
-			'altitude' => 'Altitude',
-			'location' => 'Location',
-			'image' => 'Image',
-			'video' => 'Video',
-			'size' => 'Size',
-			'format' => 'Format',
-			'resolution' => 'Resolution',
-			'duration' => 'Duration',
-			'fps' => 'Frame rate',
-			'tags' => 'Tags',
-			'camera' => 'Camera',
-			'captured' => 'Captured',
-			'make' => 'Make',
-			'type' => 'Type/Model',
-			'lens' => 'Lens',
-			'shutter' => 'Shutter Speed',
-			'aperture' => 'Aperture',
-			'focal' => 'Focal Length',
-			'iso' => 'ISO %s',
-		],
-
-		'edit' => [
-			'set_title' => 'Set Title',
-			'set_description' => 'Set Description',
-			'set_license' => 'Set License',
-			'no_tags' => 'No Tags',
-			'set_tags' => 'Set Tags',
-			'set_created_at' => 'Set Upload Date',
-			'set_taken_at' => 'Set Taken Date',
-			'set_taken_at_info' => 'When set, a star %s will be displayed to indicate that this date is not the original EXIF date.<br>Untick the checkbox and save to reset to the original date.',
-		],
-	],
-
-	'nsfw' => [
-		'header' => 'Sensitive content',
-		'description' => 'This album contains sensitive content which some people may find offensive or disturbing.',
-		'consent' => 'Tap to consent.',
-	],
-
-	'menus' => [
-		'star' => 'Star',
-		'unstar' => 'Unstar',
-		'star_all' => 'Star Selected',
-		'unstar_all' => 'Unstar Selected',
-		'tag' => 'Tag',
-		'tag_all' => 'Tag Selected',
-		'set_cover' => 'Set Album Cover',
-		'remove_header' => 'Remove Album Header',
-		'set_header' => 'Set Album Header',
-		'copy_to' => 'Copy to …',
-		'copy_all_to' => 'Copy Selected to …',
-		'rename' => 'Rename',
-		'move' => 'Move',
-		'move_all' => 'Move Selected',
-		'delete' => 'Delete',
-		'delete_all' => 'Delete Selected',
-		'download' => 'Download',
-		'download_all' => 'Download Selected',
-		'merge' => 'Merge',
-		'merge_all' => 'Merge Selected',
-
-		'upload_photo' => 'Upload Photo',
-		'import_link' => 'Import from Link',
-		'import_dropbox' => 'Import from Dropbox',
-		'new_album' => 'New Album',
-		'new_tag_album' => 'New Tag Album',
-		'upload_track' => 'Upload track',
-		'delete_track' => 'Delete track',
-	],
-
-	'sort' => [
-		'photo_select_1' => 'Upload Time',
-		'photo_select_2' => 'Take Date',
-		'photo_select_3' => 'Title',
-		'photo_select_4' => 'Description',
-		'photo_select_6' => 'Star',
-		'photo_select_7' => 'Photo Format',
-		'ascending' => 'Ascending',
-		'descending' => 'Descending',
-		'album_select_1' => 'Creation Time',
-		'album_select_2' => 'Title',
-		'album_select_3' => 'Description',
-		'album_select_5' => 'Latest Take Date',
-		'album_select_6' => 'Oldest Take Date',
-	],
-
-	'albums_protection' => [
-		'private' => 'private',
-		'public' => 'public',
-		'inherit_from_parent' => 'inherit from parent',
-	],
+    'title' => 'Galerie',
+
+    'smart_albums' => 'Albums intelligents',
+    'albums' => 'Albums',
+    'root' => 'Albums',
+
+    'original' => 'Original',
+    'medium' => 'Moyen',
+    'medium_hidpi' => 'Moyen HiDPI',
```

Comment:
```suggestion
    'medium_hidpi' => 'Moyen Haute Resolution',
```
(j'ai pas les accents sur mon clavier)